src/index.js

```diff
@@ -60,6 +61,8 @@ const NativeTachyons = {
         _.assign(styleSheet, text)
         _.assign(styleSheet, opacity)
 
+        const typeScale = generateTypeScale(options.typeScale)
```

Comment:
💁 Generates the typescale using the merged `defaultTypeScale` and passed in `typeScale` configuration.